modules/javafx.controls/src/test/java/test/javafx/scene/control/ChoiceBoxTest.java

```diff
@@ -72,6 +71,14 @@
     @Before public void setup() {
         //This step is not needed (Just to make sure StubToolkit is loaded into VM)
         tk = (StubToolkit)Toolkit.getToolkit();
+
+        Thread.currentThread().setUncaughtExceptionHandler((thread, throwable) -> {
+            if (throwable instanceof RuntimeException) {
+                throw (RuntimeException) throwable;
+            } else {
+                Thread.currentThread().getThreadGroup().uncaughtException(thread, throwable);
+            }
+        });
```

Comment:
setting the uncaughtHandler should be the very first thingy to do in before, and removing it should be the very last to do in after (which is missing here completely, must be added)